piccolo_admin/example/forms/csv.py

```diff
@@ -0,0 +1,46 @@
+import csv
+import io
+
+from pydantic import BaseModel
+from starlette.requests import Request
+
+from piccolo_admin.endpoints import FileResponse, FormConfig
+from piccolo_admin.example.tables import Movie
+
+
+class DownloadMoviesModel(BaseModel):
+    director_name: str
```

Comment:
Just a reminder. Should we add a field validator, that director name must not be empty? Something like this.

```python
class DownloadMoviesModel(BaseModel):
    director_name: str

    @field_validator("director_name")
    def validate_director_name(cls, v):
        if v == "":
            raise ValueError("it cannot be empty")
        return v
```

Without that if an empty string is entered in the form, Piccolo admin downloads an empty csv file

Reply:
OK, I forgot about that - will change it now.

Reply:
Done